tests/modules/test_combineobjects.py

```diff
@@ -39,6 +51,13 @@ def objects_y():
 
     return segmented
 
+@pytest.fixture
+def objects_y_volume():
```

Comment:
These appear to be identical to `objects_x` and `objects_x_volume`, so we might just want to have a single objects_volume fixture. These scripts were originally based off another module's so this area could probably use some cleanup.